dockerfiles/zig-0.11.Dockerfile

```diff
@@ -6,5 +6,5 @@ RUN echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/ap
 # Update the package list and install Zig
 RUN apk add --no-cache zig@testing=0.11.0-r3
 
-RUN echo "cd \${CODECRAFTERS_SUBMISSION_DIR} && zig build-exe ./app/main.zig" > /codecrafters-precompile.sh
+RUN echo "cd \${CODECRAFTERS_SUBMISSION_DIR} && zig build-exe ./src/main.zig" > /codecrafters-precompile.sh
```

Comment:
@glokta1 okay my bad I meant that we should rename to the file that we're actually using. This will break all old repositories, so we can't do that. Can you try to limit changes to only remove "duplicate" files which we aren't using?